x-pack/legacy/plugins/lens/public/indexpattern_plugin/dimension_panel/_popover.scss

```diff
@@ -0,0 +1,30 @@
+.lnsConfigPanel__summaryPopoverLeft,
+.lnsConfigPanel__summaryPopoverRight {
+  padding: $euiSizeS;
+}
+
+.lnsConfigPanel__summaryPopoverLeft {
+  padding-top: 0;
+  background-color: $euiColorLightestShade;
+}
+
+.lnsConfigPanel__summaryPopoverRight {
+  width: $euiSize * 20;
+}
+
+.lnsConfigPanel__fieldOption--incompatible {
+  color: $euiColorLightShade;
+}
+
+.lnsConfigPanel__operation {
+  padding: $euiSizeXS;
+  font-size: 0.875rem;
+}
+
+.lnsConfigPanel__operation--selected {
+  background-color: $euiColorLightShade;
+}
+
+.lnsConfigPanel__operation--incompatible {
+  opacity: 0.7;
```

Comment:
The lack of contrast here makes this a lot less effective than I was expecting. I think @cchaos might be able to make some suggestions, but I'm thinking that instead of opacity you could use a black font color for compatible, and gray font color for incompatible? 
<img width="147" alt="Screenshot 2019-06-25 13 01 58" src="https://user-images.githubusercontent.com/666475/60118055-7bfb8a00-9749-11e9-8fc1-ac37a580b332.png">

Reply:
Yeah, also not really happy with that. Maybe a separator of some kind would be a good idea?

Reply:
Let's not block the change on this